stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/PhoneNumberController.kt

```diff
@@ -61,19 +59,13 @@ class PhoneNumberController constructor(
         initiallySelectedCountryCode
     )
 
-    private val phoneNumberFormatter = countryDropdownController.selectedIndex.map {
+    private val phoneNumberFormatter = countryDropdownController.selectedIndex.mapAsStateFlow {
         PhoneNumberFormatter.forCountry(
             countryConfig.countries[it].code.value
         )
-    }.stateIn(
```

Comment:
What's the benefit over stateIn here? is it saving us a scope param?

Reply:
`selectedIndex` is now a `StateFlow` instead of a `Flow`. Therefore, we don’t need to do the Flow-to-StateFlow conversion that `stateIn` does.